SUILesson1/ContentView.swift

```diff
@@ -7,15 +7,105 @@
 
 import SwiftUI
 
+/// Представление с контентом экрана мини приложения
 struct ContentView: View {
+
+    private enum Constants {
+        static let defaultLabelText = "Текст лейбла для замены"
+        static let showButtonText = "Показать"
+        static let alertExampleText = "Пример алерта"
+        static let sayAText = "Сказать А"
+        static let sayAReplicText = "Аааааааа"
+        static let sayBText = "Сказать Б"
+        static let sayBReplicText = "Б-б-б"
+        static let showActionSheetText = "Показать шторку"
+        static let actionsText = "Действия"
+        static let actionsExemples = "Пример ActionSheet"
+        static let defaultEmptyText = ""
+    }
+
+    // MARK: - private properties
+
+    @State private var isErrorFirst = false
+    @State private var isErrorSecond = false
+    @State private var isPresented = false
+    @State private var labelText = Constants.defaultLabelText
+
+    // MARK: - public properties
+
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+
+            Form {
+                Text(labelText)
+            }
+
+            Form {
+                makeFirstButton()
+            }
+
+            Form {
+                makeButtonWithDoubleButtonAlert()
+            }
+
+            Form {
+                makeActionSheertButton()
+            }
+
+            Form {
+                makeBottomActionSheerButton()
+            }
         }
-        .padding()
+    }
+
+    private func makeFirstButton() -> some View {
+        Button(
+            action: {
+                self.isErrorFirst = true
+            },
+            label: {
+                Text(Constants.showButtonText)
+            })
+        .alert(isPresented: $isErrorFirst) {
+            Alert(title: Text(Constants.alertExampleText))
+        }
+    }
+
+    private func makeButtonWithDoubleButtonAlert() -> some View {
+        Button(
+            action: { self.isErrorSecond = true},
+            label: { Text(Constants.showButtonText)})
+        .alert(isPresented: $isErrorSecond) { Alert(
+            title: Text(Constants.alertExampleText),
+            primaryButton: .default( Text(Constants.sayAText),
+                                     action: {labelText = Constants.sayAReplicText}),
+            secondaryButton: .default(Text(Constants.sayBText),
+                                      action: { labelText = Constants.sayBReplicText}))}
```

Comment:
```suggestion
        Button { 
        self.isErrorSecond = true
        } label: { 
        Text(Constants.showButtonText)
        }
        .alert(isPresented: $isErrorSecond) { Alert(
            title: Text(Constants.alertExampleText),
            primaryButton: .default( Text(Constants.sayAText),
                                     action: {labelText = Constants.sayAReplicText}),
            secondaryButton: .default(Text(Constants.sayBText),
                                      action: { labelText = Constants.sayBReplicText}))}
```

в остальных кнопках тоже можешь так отформатировать, если понравится

Reply:
Не всюду из за вложености получается красиво. Попробую да